wellcomecollection/catalogue-api
# upgrade artillery and aws-sdk + use node20

## What does this change?

Part of https://github.com/wellcomecollection/platform/issues/5796
https://github.com/wellcomecollection/catalogue-api/security/dependabot/25

This upgrades artillery and related plugins to address vulnerability issues in one of the dependencies of `artillery-plugin-expect` 
Bonus: node20
Both stress and smoke tests ran successfully locally 

### Checklist


[swagger]: https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/main/reference/catalogue.yaml

## How to test

First: nvm use 20
Follow the instructions in catalogue-api/smoke_tests/README.md
Do not run `yarn stressTestCatalogueApiProd` as running the stage one is enough to confirm that this works. No need to risk  putting undue load on the production API

## How can we measure success?

One fewer critical vulnerability in the catalogue-api

## Have we considered potential risks?

If this happens to fail in the CI pipeline, it could block deployments of the catalogue-api
